app/src/main/java/com/keylesspalace/tusky/util/MediaUtils.kt

```diff
@@ -179,12 +179,10 @@ fun deleteStaleCachedMedia(mediaDirectory: File?) {
         return
     }
 
-    val twentyfourHoursAgo = Calendar.getInstance()
-    twentyfourHoursAgo.add(Calendar.HOUR, -24)
-    val unixTime = twentyfourHoursAgo.timeInMillis
+    val unixTime = System.currentTimeMillis() - 24.hours.inWholeMilliseconds
```

Comment:
I just simplified this code a bit (no need to instantiate a `Calendar` object to substract hours).